cmd/manager/main.go

```diff
@@ -85,7 +88,18 @@ func initStaticDeps(mgr manager.Manager) {
 		glog.Fatalf("Could not create config watch: %v", err)
 	}
 
+	bootstrapKubeconfig, err := ioutil.ReadFile(*bootstrapKubeconfig)
```

Comment:
Can we skip trying to load this if `*bootstrapKubeconfig == ""` ?  (because otherwise we'll spew a warning)